utils/build/docker/java/install_ddtrace.sh

```diff
@@ -4,6 +4,19 @@ set -eu
 
 mkdir /dd-tracer
 
+# Look for custom dd-trace-api jar in custom binaries folder
+CUSTOM_DD_TRACE_API_COUNT=$(find /binaries/dd-trace-api*.jar 2>/dev/null | wc -l)
+if [ "$CUSTOM_DD_TRACE_API_COUNT" = 0 ]; then
+    echo "Using default dd-trace-api"
```

Comment:
Could we default to latest, rather than a pinned version?

Reply:
This is just the script to mount a custom `dd-trace-api.jar` into the docker container. What do you mean by pinned version?

Reply:
In the `pom.xml` of each weblog we are defining the version of the dd-trace-api as:

```
<dd-trace-api.version>[1.47.0,)</dd-trace-api.version>
```

So in the absence of a custom `dd-trace-api.jar` in the binaries folder, it will download the latest snapshot version from sonatype.